roles/deploy/tasks/main.yml

```diff
@@ -27,6 +27,11 @@
       repo: "{{ repo }}"
       dest: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sqlite dependencies
```

Comment:
If you're not using the deploy role, you don't need it. We don't use sqlite in production, but when you bundle install the dev environment in the ssh user's home directory, you need to install the dev gems to get capistrano. I suppose we could edit the Gemfile and create a "deploy" group, then we'd be able to bundle install without dev/test in this role and we wouldn't need the sqlite headers.